RecoLocalCalo/Configuration/python/RecoLocalCalo_cff.py

```diff
@@ -31,3 +31,38 @@
 #
 from RecoLocalCalo.Configuration.hcalLocalRecoNZS_cff import *
 calolocalrecoNZS = cms.Sequence(ecalLocalRecoSequence+hcalLocalRecoSequence+hcalLocalRecoSequenceNZS) 
+
+def _modifyRecoLocalCaloConfigurationReconstructorsForPhase2CommonHBHE( obj ):
+    obj.digiLabel = cms.InputTag('simHcalDigis','HBHEUpgradeDigiCollection')
+
+def _modifyRecoLocalCaloConfigurationReconstructorsForPhase2CommonHO( obj ):
+    obj.digiLabel = cms.InputTag('simHcalDigis')
+
+def _modifyRecoLocalCaloConfigurationReconstructorsForPhase2CommonHF( obj ):
+    obj.digiLabel = cms.InputTag('simHcalDigis','HFUpgradeDigiCollection')
+
+def _modifyRecoLocalCaloConfigurationReconstructorsForPhase2CommonZDC( obj ):
+    obj.digiLabel = cms.InputTag('simHcalUnsuppressedDigis')
+
+def _modifyRecoLocalCaloConfigurationProcessForPhase2Common( theProcess ):
+    theProcess.load("RecoLocalCalo.HcalRecProducers.HBHEUpgradeReconstructor_cfi")
+    theProcess.load("RecoLocalCalo.HcalRecProducers.HFUpgradeReconstructor_cfi")
```

Comment:
@slava77 - it was agreed in the [discussion](https://github.com/cms-sw/cmssw/pull/13992/files/45d26ee968bca52533a24dd8974503373e3e9548#r59400120) with @Dr15Jones that HCAL will take care of the hardcode condition loading later, in a subsequent PR. It is not worth holding up this one.